controllers/labels.go

```diff
@@ -0,0 +1,5 @@
+package controllers
```

Comment:
Please don't keep a commit with my name on it. These fixes should be distributed in their relevant commits.